css/print.css

```diff
@@ -84,119 +528,123 @@ emu-figure img {
 }
 
 #toc > ol > li {
-  margin-top: 1ex;
+  margin-top: 0.75ex;
 }
 
 #toc,
 #spec-container > emu-intro,
 #spec-container > emu-annex {
   break-before: recto;
-  break-after: always;
+  break-after: page;
 }
 
-/* according to Ecma guidelines, we're actually not supposed to break before every clause (only the first), but Paged.js fails if we do that */
-/* so instead, just break before any of the clauses that have sub-clauses */
-#spec-container > emu-clause:has(emu-clause:not([example])) {
-  break-before: always;
+a[data-print-href]::after {
+  content: ' <' attr(href) '>';
+  color: initial;
 }
 
-#spec-container > emu-clause:first-of-type {
-  break-before: recto;
+.real-table {
+  max-width: 100%;
+  width: auto;
 }
 
-emu-note,
-emu-note p,
-emu-table tr,
-emu-table th,
-emu-table td,
-emu-alg li,
-pre,
-h1,
-#metadata-block {
-  break-inside: avoid;
+.annex-title {
+  text-align: center;
 }
 
-emu-table thead,
-h1,
-figcaption,
-emu-alg > ol > li:first-child {
-  break-after: avoid;
+.copyright-notice { /* ecma mandated */
+  font-style: italic;
+  border: 1px solid black;
+  padding: 1em;
+  page: copyright;
+  page-break-before: always;
+  page-break-after: always;
 }
 
-emu-grammar + emu-alg,
-figcaption + emu-table,
-figcaption + span[id] + emu-table,
-emu-alg > ol > li:last-child {
-  break-before: avoid;
+.full-page-svg {
+  color: rgba(255, 255, 255, 0);
 }
```

Comment:
@gesa Kevin and I couldn't figure out what this class is used for. Why have transparent white text on the front cover and inner cover? And what's with the class name?